driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -144,6 +146,18 @@ public void save(T entity) {
     }
 
     /**
+     * Save an entity mapped by this mapper and using special options for save.
+     * <p/>
+     *
+     * @param entity the entity to save.
+     * @param options the options object specified defining special options when saving.
+     */
+    public void save(T entity, QueryType.SaveOptions options) {
+        this.saveOptions = options;
```

Comment:
No, see Mapper.java (l.130), and `MapperSaveOptionsTest#should_use_using_options_only_once` .